unit_tests/CMakeLists.txt

```diff
@@ -1,13 +1,13 @@
 cmake_minimum_required(VERSION ${CMAKE_VERSION})
 project(unit_tests LANGUAGES C CXX)
 
-set(BUILD_UNIT_TESTS YES CACHE BOOL "Build unit tests")
-set(UNIT_TESTS_RUN_AFTER_BUILD NO CACHE BOOL "Run after building unit tests")
-set(UNIT_TESTS_REPORTING_STYLE "junit" CACHE STRING "The style of output used for unit test reporting [console, compact, junit, xml]")
-set(UNIT_TESTS_REPORT_FILENAME "report.xml" CACHE STRING "The filename of the unit test report")
-set(UNIT_TESTS_ENABLE_ALL NO CACHE BOOL "Enables all unit tests")
+set(IRODS_BUILD_UNIT_TESTS NO CACHE BOOL "Build unit tests")
```

Comment:
How do we feel about `IRODS_UNIT_TESTS_BUILD` and `IRODS_UNIT_TESTS_RUN_AFTER_CORE_BUILD`?

Reply:
I think the suggestions make sense.

The suggested name for the post build option is incorrect though. That option means **run that specific test once it's compiled**. That option doesn't take into account that the packages may not be installed. For that reason, I feel that option needs to be removed.

Reply:
Does anyone else have thoughts about the post build option?

Reply:
running pre-install doesn't make any sense - can remove them.

Reply:
done.